src/style.css

```diff
@@ -0,0 +1,102 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.html {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.form {
+  margin: 20px 0;
+  border: solid 1px #e1e7ed;
+  border-radius: 4px;
+  height: 70px;
+  position: relative;
+  margin-top: 20px;
+}
+
+.second--form {
+  height: 42px;
+  position: relative;
+}
+
+.input {
+  font-family: inherit;
+  font-weight: 300;
+  border: 0;
+  padding: 0;
+  height: 100%;
+  width: 100%;
+  color: #3d4e61;
+  outline: none;
+  background: none;
+  font-size: 16px;
+  padding-left: 62px;
+}
+
+.input::placeholder {
+  color: #3d4e61;
+  font-family: inherit;
+  font-weight: 300;
+}
+
+.input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input:focus {
+  background: linear-gradient(#fff 0%, #f6f6f7 100%);
+  font-family: inherit;
+  font-weight: 800;
+  border-radius: 3px;
+}
+
+#search-input::placeholder {
+  font-size: 16px;
+}
+
+#search-input:focus {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  text-shadow: 0 3px 3px rgba(0, 0, 0, 0.25);
+}
+
+.second--input {
+  font-size: 14px;
+  padding-left: 33px;
+}
+
+.second--input::placeholder {
+  font-size: 14px;
+}
+
+.image {
+  position: absolute;
+}
+
+#img1 {
+  top: 25px;
+  left: 26px;
+}
+
+.image--2 {
+  top: 15px;
+  left: 13px;
+  width: 11px;
+  height: 11px;
+}
```

Comment:
try to come up with the solution where you have main class .img and then more specific .img-small
![Screenshot 2023-03-03 at 17 29 25](https://user-images.githubusercontent.com/6526149/222760316-84cf5c82-9cf4-4559-bba2-0deebb032d15.png)


```suggestion
.image {
  ....
}

.image--small {
  ...
}
```